projects/aws/eks-anywhere-test/eks-anywhere-test/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: eks-anywhere-test
 description: EKS Anywhere Packages Test Application
 type: application
-version: {{IMAGE_TAG}}-helm
-appVersion: {{IMAGE_TAG}}
+version: {{IMAGE_TAG}}
+appVersion: {{IMAGE_TAG}}-helm
```

Comment:
Is this correct? Seems like we should just remove `-helm` here, remove the `sed` line that removes it, and not add `-helm` to appVersion.

Reply:
Will do.